src/olympia/blocklist/tests/test_cron.py

```diff
@@ -24,11 +25,16 @@
 from olympia.zadmin.models import get_config, set_config
 
 
+STATSD_PREFIX = 'blocklist.cron.upload_mlbf_to_remote_settings.'
+
+
 @freeze_time('2020-01-01 12:34:56')
 @override_switch('blocklist_mlbf_submit', active=True)
 class TestUploadToRemoteSettings(TestCase):
     def setUp(self):
-        addon_factory()
+        addon = addon_factory()
+        version_factory(addon=addon)
+        version_factory(addon=addon)
```

Comment:
Is this meant to be called twice?

Reply:
yeah, it's generating 2 extra versions (so addon will have 3 versions total).  Doesn't matter much for the bloomfilter itself, but it was useful to see different numbers in the statsd calls.)